appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/__tests__/lpa-questionnaire.test.js

```diff
@@ -1226,6 +1213,27 @@ describe('LPA Questionnaire review', () => {
 			expect(element.innerHTML).toMatchSnapshot();
 		});
 
+		it('should re-render the document details page with the expected error message if receivedDate is a date in the future', async () => {
+			const today = new Date();
+			const response = await request.post(`${baseUrl}/add-document-details/1`).send({
+				items: [
+					{
+						documentId: 'a6681be2-7cf8-4c9f-b223-f97f003577f3',
+						receivedDate: {
```

Comment:
This may fail at some point in the future - e.g. 30 Jan 2024 -> 31 Feb 2025?

Reply:
Thanks, it's a good point - I've updated this code to use date-fns to generate a future date, hopefully more robust!